.golangci.yml

```diff
@@ -38,81 +55,69 @@ linters:
     - musttag
     - nakedret
     - nilerr
+    - noctx
+    - nolintlint
     - nosprintfhostport
     - perfsprint
     - prealloc
     - predeclared
     - promlinter
     - protogetter
     - reassign
+    - rowserrcheck
     - sloglint
     - spancheck
+    - sqlclosecheck
```

Comment:
Ditto as https://github.com/kubernetes-sigs/cri-tools/pull/1496/files#r1680376638.

Reply:
I'd say leave it enabled if we either add code which the linter may catch in the future.